### PR TITLE
User doc instructions for how to set container path

### PR DESCRIPTION
**Pull request by thenor**
Signed-off-by: Therese Nordqvist <therese.nordqvist@pelagicore.com>

### CODE REVIEW

File: doc/chapters/configuration/index.rst [hunk 1]
```diff
@@ -73,40 +73,46 @@ Command line options
...
+is configured and installed by CMake. This is passed to LXC by SC.
 
-The other source of LXC specific configuration is in ``libsoftwarecontainer/softwarecontainer.conf.in`` which is
-configured and installed by CMake to |lxc-config-path-code|.
+The other source of LXC specific configuration is in ``libsoftwarecontainer/softwarecontainer.conf.in``
+which is configured and installed by CMake to |lxc-config-path-code|.
+
+LXC has other options which can effect how SoftwareContainer behaves. One such example is the
+default path to use for containers. This option can be configured by setting ``lxc.lxcpath`` in
+the LXC configuration file (typically found in ``/etc/lxc/lxc.conf`` or ``/usr/etc/lxc/lxc.conf``).
```
Comment (DunderRoffe) [inline doc/chapters/configuration/index.rst]: I would prefer to have this as a bullet list instead. It is easier to read when scrolling thorough the text and has better support for more instances.

**Review by DunderRoffe — CHANGES_REQUESTED** [review]

**Review by tobsan — APPROVED** [review]

File: doc/chapters/configuration/index.rst (hunk 1 above)
Comment (thenor) [reply doc/chapters/configuration/index.rst]: I will re-write it and make a table so it's more clear.

File: doc/chapters/configuration/index.rst [hunk 2]
```diff
@@ -73,40 +73,55 @@ Command line options
...
 
-LXC is configured partly by the LXC template ``libsoftwarecontainer/lxc-softwarecontainer.in`` which is configured
-and installed by CMake. This is passed to LXC by SC.
+LXC is configured partly by the LXC template ``libsoftwarecontainer/lxc-softwarecontainer.in`` which
+is configured and installed by CMake. This is passed to LXC by SC.
+
+Another source of LXC specific configuration is in ``libsoftwarecontainer/softwarecontainer.conf.in``
+which is configured and installed by CMake to |lxc-config-path-code|.
+
+A third souce is the LXC config file, typically found in ``/etc/lxc/lxc.conf`` or
```
Comment (dzudek) [inline doc/chapters/configuration/index.rst]: Typo "souce"

File: doc/chapters/configuration/index.rst [hunk 3]
```diff
@@ -73,40 +73,55 @@ Command line options
...
-and installed by CMake. This is passed to LXC by SC.
+LXC is configured partly by the LXC template ``libsoftwarecontainer/lxc-softwarecontainer.in`` which
+is configured and installed by CMake. This is passed to LXC by SC.
+
+Another source of LXC specific configuration is in ``libsoftwarecontainer/softwarecontainer.conf.in``
+which is configured and installed by CMake to |lxc-config-path-code|.
+
+A third source is the LXC config file, typically found in ``/etc/lxc/lxc.conf`` or
+``/usr/etc/lxc/lxc.conf``.
+The following is a list of LXC config items which can be used to configure SoftwareContainer.
```
Comment (tobsan) [inline doc/chapters/configuration/index.rst]: This isn't exactly true. There are shitloads of config items.

Perhaps one can mention that these are the ones we feel are relevant currently.

**Review by DunderRoffe — APPROVED** [review]

**Review by tobsan — APPROVED** [review]

**Pull request merged**